tests/kernel/mem_protect/userspace/src/main.c

```diff
@@ -964,15 +967,19 @@ void test_tls_leakage(void)
 
 	k_thread_user_mode_enter(tls_leakage_user_part,
 				 _current->userspace_local_data, NULL, NULL);
+#endif
```

Comment:
`ztest_test_skip` should be called here so that the test is shown as "skipped" and not "succeeded."

```c
#else
	ztest_test_skip();
```

Reply:
Ah, *that's* how you do that :-)